### PR TITLE
Ag/dmpc trajectory generator

### PR DESCRIPTION
**Pull request by gstomberg**

### CODE REVIEW

File: holohover_dmpc/src/trajectory_generator.cpp [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    }
+
+    return gc;
+}
+
+int main(int argc, char** argv)
+{
+    rclcpp::init(argc, argv);
+    auto node = std::make_shared<TrajectoryGenerator>();
+    std::this_thread::sleep_for(std::chrono::seconds(3)); //wait for publishers to be setup
```
Comment (RSchwan) [inline holohover_dmpc/src/trajectory_generator.cpp:105]: I think this should be done in the launch file. There is a `TimerAction` for this. Take a look [here](https://github.com/RSchwan/holohover/blob/main/holohover_utils/launch/common/recorder.launch.py#L26) for an example.

Comment (gstomberg) [reply holohover_dmpc/src/trajectory_generator.cpp:105]: Do you suggest to keep the sleep_for and pass the "3" in the above example programmatically via a launch file? Or does the launch file remove the need for sleep_for?

Comment (RSchwan) [reply holohover_dmpc/src/trajectory_generator.cpp:105]: The launch file removes the need for sleep_for

Comment (gstomberg) [reply holohover_dmpc/src/trajectory_generator.cpp:105]: We start the trajectory generator with this command: ros2 run holohover_dmpc trajectory_generator --ros-args -p trajectory_file:=[path to the trajectory file]. How can I pass the trajectory_file via launch file? I am trying 

trajectory_config = os.path.join(
        get_package_share_directory('holohover_dmpc'),
        'config/trajectories',
        'trajectory1.yaml'
    )

trajectory_node = Node(
        package="holohover_dmpc",
        executable="trajectory_generator",
        arguments=["trajectory_file",trajectory_config],
        output='screen'
    )

Comment (gstomberg) [reply holohover_dmpc/src/trajectory_generator.cpp:105]: Got it: 
trajectory_node = Node(
        package="holohover_dmpc",
        executable="trajectory_generator",
        parameters=[{"trajectory_file": trajectory_config}] ,
        output='screen'
    )

**Pull request merged**